Fix:
error: implicit declaration of function 'execve' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'setgroups' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'setgid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'setuid' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'lseek' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- src/pathexec_run.c.orig	2008-06-15 15:31:14.000000000 -0500
+++ src/pathexec_run.c	2022-01-17 05:59:24.000000000 -0600
@@ -1,5 +1,6 @@
 /* Public domain. */
 
+#include <unistd.h>
 #include "error.h"
 #include "stralloc.h"
 #include "str.h"
--- src/prot.c.orig	2008-06-15 15:31:14.000000000 -0500
+++ src/prot.c	2022-01-17 06:03:24.000000000 -0600
@@ -1,5 +1,6 @@
 /* Public domain. */
 
+#include <unistd.h>
 #include "hasshsgr.h"
 #include "prot.h"
 
--- src/seek_set.c.orig	2008-06-15 15:31:14.000000000 -0500
+++ src/seek_set.c	2022-01-17 06:05:46.000000000 -0600
@@ -1,6 +1,7 @@
 /* Public domain. */
 
 #include <sys/types.h>
+#include <unistd.h>
 #include "seek.h"
 
 #define SET 0 /* sigh */
